src/jvm/test/resources/expected_logs/run_concurrent_test/kotlin_thread.txt

```diff
@@ -0,0 +1,42 @@
+= Concurrent test failed =
```

Comment:
Do we need this line, or should we start the output with the exception?

Reply:
I would leave it. 

1) It is a title of the whole report, explaining what happened. Starting with the exception might be confusing, as if the code itself thrown an exception.

2) Currently there are two possible lines:

```
= Concurrent test failed =
```


```
= Concurrent test has hung =
```